pkg/forwarder/forwarder_health.go

```diff
@@ -38,7 +40,7 @@ var (
 
 func init() {
 	apiKeyStatusUnknown.Set("Unable to validate API Key")
-	apiKeyInvalid.Set("API Key invalid")
+	apiKeyInvalid.Set(color.YellowString("API Key invalid"))
```

Comment:
Why not red?

Reply:
Hmm - are we supposed to assume that an expvar is always going to a TTY since we're using TTY escape codes? My assumption is that this may work only when using `agent status` and may somewhat garble any other expvar UI, including any expvar logic that expects a plain string. If that's the case, we may want to consider not colorizing the expvar value itself here but doing some conditional processing in the `agent status` command itself.

PS: +1 on @djmitche's comment from me as well

Reply:
the agent status is copied in a flare. Im not sure if these escape codes are scrubbed, but it may be worth checking.